src/main/routes/task-list.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express'
+import { SubTaskListBuilder } from '../views/macros/elements/task-list/util/subTaskListBuilder';
+
+const router = express.Router()
+
+router.get('/task-list', (req, res, next) => {
+  res.render('task-list', {
+    tasks: {
+      applicantDetails: SubTaskListBuilder.buildApplicationDetailsSubTasks(),
+      yourDetails: SubTaskListBuilder.buildYourDetailsSubTasks(),
+      childDetails: SubTaskListBuilder.buildChildDetailsSubTasks(),
+      adoptionContacts: SubTaskListBuilder.buildAdoptionContactsSubTasks(),
+      declarePayments: SubTaskListBuilder.buildDeclarePaymentsSubTasks(),
+      reviewApplication: SubTaskListBuilder.buildReviewApplicationSubTasks()
+    }
+  })
+})
```

Comment:
@KiranVarma02 Built a TaskListBuilder for this but I found with the embedded info components in some of our tasks that it becomes very fragile. We can discuss. For now decided to go with preBuilding subTasks.